scripts/publish-npm-packages.sh

```diff
@@ -44,3 +44,17 @@ echo "$registry_without_protocol/:_authToken=${NPM_TOKEN}" >> ~/.npmrc
 for file in ./npm-artifacts/*.tgz; do
     npm publish "$file" --tag "$dist_tag" --registry "$registry"
 done
+
+# Check if any files in packages/grafana-e2e-selectors were changed. If so, add a 'modified' tag to the package
+COUNT=$(git diff HEAD~1..HEAD --name-only -- packages/grafana-e2e-selectors | awk 'END{print NR}')
+if (( $COUNT > 0 )); then
+    # Wait a little bit to allow the package to be published to the registry
+    sleep 5s
+    regex_pattern="canary: ([0-9.-]+)"
+    TAGS=$(npm dist-tag ls @grafana/e2e-selectors)
+    if [[ $TAGS =~ $regex_pattern ]]; then
+        echo "$COUNT file(s) in packages/grafana-e2e-selectors were changed. Adding 'modified' tag to @grafana/e2e-selectors@${BASH_REMATCH[1]}"
+        npm dist-tag add @grafana/e2e-selectors@${BASH_REMATCH[1]} modified
```

Comment:
I was wondering if a dist-tag like `alpha` would suite better? Just an idea.

Reply:
That's a good option too, thanks! Once the enterprise datasources stop using the canary releases on the e2e selectors in their ci, my hope is that we can remove this new dist tag and simply _only_ release this package once its actually changed.